lib/internal/inspector/inspect_client.js

```diff
@@ -198,11 +180,11 @@ class Client extends EventEmitter {
       debuglog('< %s', payloadStr);
       const lastChar = payloadStr[payloadStr.length - 1];
       if (payloadStr[0] !== '{' || lastChar !== '}') {
-        throw new Error(`Payload does not look like JSON: ${payloadStr}`);
+        throw new ERR_DEBUGGER_ERROR(`Payload does not look like JSON: ${payloadStr}`);
```

Comment:
@Trott  This line over 80 chars comes from https://github.com/nodejs/node/pull/39024/commits/c22eec85b477f73e95235caadbcaa2bb433e5f49, do you know why this wasn't an issue on `master`?

Reply:
On master, we relaxed the line length lint rule for lines containing template strings because splitting them up across multiple lines often (usually) inhibited readability rather than helping it.

Reply:
Specifically: 038608d40105db8447afdf8a1b823eafe98b8a75